app/Models/Task.php

```diff
@@ -44,4 +44,14 @@ public function getAllTasks()
     {
         return $this->all();
     }
+
+    public function deleteTasks($id)
+    {
+        $idTasks = $this->find($id);
```

Comment:
tìm k có thì thế nào?

Reply:
Dạ, nếu không tìm thấy sẽ  báo lỗi Trying to get property 'st' of non-object

Reply:
xử lý lại hết như a nói ở trên